index.html

```diff
@@ -16,11 +16,11 @@
       <div class="menu-bar">
         <a href="welcome" class="head">Welcome</a>
         <nav class="menu-bar2">
-          <ul class="nav-menu">
+          <ul class="nav-menu" id="menu">
```

Comment:
Good job with the animation, it looks good. But there are some changes needed to be made. 

- Firstly please add some space before the navbar. There is almost no place above the nav. 
<img width="517" alt="Screenshot 2023-03-13 at 9 58 33 PM" src="https://user-images.githubusercontent.com/25887253/224805858-94cda9ec-5a86-424d-b93e-4ae72610f4d9.png">
<img width="392" alt="Screenshot 2023-03-13 at 10 12 21 PM" src="https://user-images.githubusercontent.com/25887253/224806415-a2f7faba-d1f9-4ac9-93bc-87032ba6fed4.png">

- Kindly make sure you are following the Figma template. On 375px it looks like the below, it is out of view. And there is a horizontal overflowing issue, please fix it. Check the GIF below.
<img width="281" alt="Screenshot 2023-03-13 at 9 58 54 PM" src="https://user-images.githubusercontent.com/25887253/224806826-8ae54af3-392d-43ca-9565-996c07dd5de7.png">

![Screen Recording 2023-03-13 at 9 59 13 PM](https://user-images.githubusercontent.com/25887253/224811465-59deb4d0-c8a3-4eaa-bd31-2840a24105ef.gif)




- Kindly fix the color of the close button.
<img width="505" alt="Screenshot 2023-03-13 at 10 05 45 PM" src="https://user-images.githubusercontent.com/25887253/224808539-f08583de-750c-4abf-a926-4d97d696e092.png">
<img width="505" alt="Screenshot 2023-03-13 at 10 06 03 PM" src="https://user-images.githubusercontent.com/25887253/224808545-233eeeb4-0635-4705-8d44-9e7653e773bd.png">

- Please fix the font size and the color of the links.
<img width="377" alt="Screenshot 2023-03-13 at 10 06 17 PM" src="https://user-images.githubusercontent.com/25887253/224809158-7e414733-0301-4f81-ba16-227c35e49a0e.png">
<img width="628" alt="Screenshot 2023-03-13 at 10 06 52 PM" src="https://user-images.githubusercontent.com/25887253/224809165-bfe99fd9-d627-4dd7-9d60-c868a67724be.png">

- Please make sure your nav links are aligned like the Figma template, there shouldn't be centered.
<img width="377" alt="Screenshot 2023-03-13 at 10 06 29 PM" src="https://user-images.githubusercontent.com/25887253/224809838-9a464e34-fb67-40c2-87f8-1852271ce64f.png">

- According to the requirements, when you click a link, nav should be closed. On your nav is closing. Kindly fix the issue.
![klkk](https://user-images.githubusercontent.com/25887253/224811915-7acc8b8c-eb3c-4351-a7e1-76d9cb776eb6.gif)